### PR TITLE
Admin operation file

### PR DESCRIPTION
**Pull request by ovekyc**
This file is for admin

### CODE REVIEW

File: admin.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const Commands = {
+  db: levelup(config.database, {valueEncoding: 'json'}),
+  db_get_all: function() {
+    this.db.createReadStream({
+      start: '\x00',
+      end: '\xFF'
+    })
+    .on('data', console.log)
+    .on('error', console.log);
+  },
```
Comment (hewonjeong) [inline admin.js:13]: How about using a parameter like prefix or option?
Or we can make other function to search specific entity.

**Comment by festinalente91** [comment]
lgtm

**Comment by ovekyc** [comment]
lgtm

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**